lib/auth/auth_with_roles.go

```diff
@@ -266,6 +267,12 @@ func (a *ServerWithRoles) LoginRuleClient() loginrulepb.LoginRuleServiceClient {
 	panic("LoginRuleClient not implemented by ServerWithRoles")
 }
 
+// PluginsClient allows ServerWithRoles to implement ClientI.
+// It should not be called through ServerWithRoles and will always panic.
+func (a *ServerWithRoles) PluginsClient() pluginspb.PluginServiceClient {
+	panic("PluginsClient not implemented by ServerWithRoles")
```

Comment:
This feels really bad; could we replace this one (and the other two `FooClient` functions) with a `pluginspb.PluginServiceClient` that always returns `NotImplemented`? It could be done for all of them in a single pass by implementing a noop `grpc.ClientConnInterface` that always errors out.